webapp-django/crashstats/topcrashers/templates/topcrashers/topcrashers.html

```diff
@@ -0,0 +1,197 @@
+{% extends "crashstats_base.html" %}
+{% block site_css %}
+  {{ super() }}
+
+  {% compress css %}
+  <link href="{{ static('crashstats/css/topcrashers.less') }}" rel="stylesheet" type="text/less" media="screen" />
+  {% endcompress %}
+{% endblock %}
+
+{% block page_title %}
+Top Crashers for {{ product }} {{ version }}
+{% endblock %}
+
+{% block content %}
+    <div id="mainbody">
+        <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+        <div class="page-heading">
+            <h2>Top Crashers for <span id="current-product">{{ product }}</span> <span id="current-version">{{ version }}</span></h2>
+        </div>
+
+        <div class="panel">
+            <div class="body">
+
+            {% if total_crashing_signatures %}
+            <div>
+                Top {{ total_crashing_signatures }} Crashing Signatures.
+                <time class="start-date">{{ query.start_date }}</time> through
+                <time class="end-date">{{ query.end_date }}</time>.
+                <p>The report covers <span class="percentage" title="{{ number_of_crashes }} ({{ (total_percentage * 100) | round(2) }}%)">{{ number_of_crashes }} ({{ (total_percentage * 100) | round(2) }}%)</span> of all {{ total_number_of_crashes }} crashes during this period. </p>
```

Comment:
It irks me that we have a variable called `total_percentage` but then we multiply it with 100. If it was `total_ratio * 100` or something it'd make sense.